apps/sms/test/unit/thread_ui_test.js

```diff
@@ -186,10 +186,12 @@ suite('thread_ui.js >', function() {
 
     this.sinon.stub(MessageManager, 'on');
     this.sinon.stub(Compose, 'on');
+    this.sinon.stub(Compose, 'off');
     this.sinon.useFakeTimers();
 
     ThreadUI.recipients = null;
     ThreadUI.init();
+    ThreadUI.enableConvertNoticeBanners();
```

Comment:
nit: I believe this line is only needed for 'segmentInfo management >' test suite and other tests don't need it ('message type conversion >' suite has its own), so could you please move this to [1] if I got it correctly?

[1] https://github.com/azasypkin/gaia/blob/d733b56f2bc857ed84a0411f3da0d6c5a70c0b73/apps/sms/test/unit/thread_ui_test.js#L487